ui/app/AppLayouts/Chat/stores/CommunitiesStore.qml

```diff
@@ -165,4 +167,15 @@ QtObject {
         console.log("TODO: Remove permissions - backend call")
         root.permissionsModel.remove(index)
     }
+
+    //MINTING
+
+    property var mintTokensModel: mintingModuleInst.tokensModel
```

Comment:
Could we specify the property type here? As said, there is an open discussion about how to better redefine stores but a common agreement is whether we can expose typed backend properites, more explicit, and less ambiguity.